lib/lotus/routing/http_router.rb

```diff
@@ -146,15 +146,16 @@ def no_response(request, env)
       end
 
       private
-      def add_with_request_method(path, method, opts = {}, &app)
-        super.generate(@resolver, opts, &app)
-      end
 
-      def _rescue_url_recognition
+      def _rescue_url_recognition(*args)
         yield
       rescue ::HttpRouter::InvalidRouteException,
              ::HttpRouter::TooManyParametersException => e
-        raise Routing::InvalidRouteException.new(e.message)
+        raise Routing::InvalidRouteException.new("#{ e.message }, please check your arguments")
```

Comment:
@AlfonsoUceda I'm not a native english speaker, but I think it's better `"#{ e.message } - please check given arguments"`. :smile_cat: